framework/ChiMesh/VolumeMesher/chi_volumemesher_01_utils.cc

```diff
@@ -1,6 +1,13 @@
 #include "chi_volumemesher.h"
 #include "ChiMesh/MeshContinuum/chi_meshcontinuum.h"
 
+//###################################################################
+chi_mesh::VolumeMesher::VolumeMesher(VolumeMesherType type) :
+ m_type(type)
+{
+
+}
+
```

Comment:
A lot of space to take up for some empty braces. Thoughts on putting `{ }` or `{}` on line 7?

Reply:
Done